package.json

```diff
@@ -41,8 +40,8 @@
     "react-toastify": "^5.3.1",
     "redux": "^4.0.1",
     "redux-devtools-extension": "^2.13.8",
-    "redux-mock-store": "^1.5.3",
-    "redux-thunk": "^2.3.0"
+    "redux-thunk": "^2.3.0",
+    "react-test-renderer": "^16.8.6"
```

Comment:
I am wondering why the redux-mock-store package was removed, won't it affect the functionality and tests of other features.

Reply:
The tests are running perfectly. Seems it was unnecessary.

Reply:
I understand @Manorlds-Eaglespark that tests do pass but please consider adding the package back it is being used in many other files.
![image](https://user-images.githubusercontent.com/22131999/61625202-d02f5680-ac82-11e9-8f41-f0dd7f738016.png)
![image](https://user-images.githubusercontent.com/22131999/61625362-34521a80-ac83-11e9-9f3d-0dab70f6f4f8.png)

Reply:
@Manorlds-Eaglespark kindly let's have it back in our packages, since we are using `configureMockStore` from it while testing actions